src/tests/all.rs

```diff
@@ -245,6 +246,17 @@ fn mock_keyword(req: &mut Request, name: &str) -> Keyword {
     Keyword::find_or_insert(req.tx().unwrap(), name).unwrap()
 }
 
+fn mock_category(req: &mut Request, name: &str) -> Category {
```

Comment:
ARGH it's not a mock why don't people use the right words. I'm sure this is consistent with existing though.